CHANGELOG.md

```diff
@@ -465,4 +466,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.0.3]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.2...1.0.3
 [1.0.2]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.1...1.0.2
 [1.0.1]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.0...1.0.1
-[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
+[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
```

Comment:
I'm approving this PR. 
There is some unrelated change here. Please remove it if it can be fixed. Thanks.